content/uiux/concepts/bias-in-user-research/bias-in-user-research.md

```diff
@@ -0,0 +1,111 @@
+
+<details open> 
+
+<summary>Example</summary> 
+
+<br> 
+
+This survey asked, “Which of the following emotions do you feel when at the airport? Please select the options below.”  
+
+![Bar Chart that displays the data from the survey question: Which of the following emotions do you feel at the airport?](media/confirmation-bias-chart.png) 
+
+The team’s larger goal is to make a product that boosts positive feelings at the airport. Confirmation bias might mean looking at this data and only reporting findings based on the statistic “57.1 of our participants are excited when traveling at the airport.” While this is true, it ignores the other data and emotions, that 42.9% also feel anxious, confused, and frustrated.  </details> 
+
+### Demand Characteristics Bias 
+
+__Demand characteristics bias__ is a change in how a user answers questions due to the knowledge that they are part of a research study. If the user knows, or thinks they know, the objective of a study, this could influence their responses and behaviors. 
+
+<details open> 
+
+<summary>Example</summary> 
+
+<br> 
+
+A researcher runs a concept test to evaluate an early design idea. The participant may believe that the goal is to validate the idea so the company can move on to the development phase. This belief could influence them to try to “help” the researcher. They might provide positive feedback even if they don’t like the idea. </details> 
+
+### Framing Effect 
+
+The __framing effect__ occurs when findings are presented in a way that is disingenuous. Examples of the framing effect could be excluding certain data points, using statistical measures that are skewed, or strategically phrasing an insight to imply a certain idea or opinion. Researchers should present the data in a way that accurately represents respondents’ statements. 
+
+<details open> 
+
+<summary>Example</summary> 
+
+<br> 
+
+Imagine that 20 survey respondents rated their overall approval of a certain product on a Likert scale of 1 to 5.  
+
+![Likert Scale Bar Chart that displays the data from the survey question: Approval from 1-5](media/framing-effect.png) 
+
+The researcher reports that the average approval rate is between 2 and 3, meaning that users are slightly dissatisfied. Considering that the mean is 2.45, this is technically correct. But it is a disingenuous finding &mdash; only 25% of the users actually rated the product between a 2 and a 3.  
+
+It would be more accurate to report that the majority of users are either extremely dissatisfied or extremely satisfied with the product. By reporting the finding that the approval rate is between 2 and 3, the team might miss the root of the issue that causes disparities in the satisfaction of certain groups of users. </details> 
```

Comment:
```suggestion
```html
<details open> 

<summary>Example</summary> 

<br> 

Imagine that 20 survey respondents rated their overall approval of a certain product on a Likert scale of 1 to 5.  

![Likert Scale Bar Chart that displays the data from the survey question: Approval from 1-5](media/framing-effect.png) 

The researcher reports that the average approval rate is between 2 and 3, meaning that users are slightly dissatisfied. Considering that the mean is 2.45, this is technically correct. But it is a disingenuous finding &mdash; only 25% of the users actually rated the product between a 2 and a 3.  

It would be more accurate to report that the majority of users are either extremely dissatisfied or extremely satisfied with the product. By reporting the finding that the approval rate is between 2 and 3, the team might miss the root of the issue that causes disparities in the satisfaction of certain groups of users. </details> 
```
```